src/marten/http/session/store/base.cr

```diff
@@ -9,12 +9,27 @@ module Marten
           @accessed : Bool = false
           @modified : Bool = false
           @session_hash : SessionHash? = nil
+          @expires : Int32? = nil
 
           getter session_key
 
           def initialize(@session_key : String?)
           end
 
+          def expires=(value : Int32)
+            @expires = value
+            @modified = true
+          end
+
+          def expires
+            @expires ||= Marten.settings.sessions.cookie_max_age
+          end
+
+          def expires_at
+            return nil if expires == 0
```

Comment:
We shouldn't allow for non-expiring sessions. Can we remove this from this PR? 🙏 

If users want to set a very big expiry time by default that's fine, but I don't think the framework should allow never-expiring sessions as this is a very bad practice.

Reply:
For cookies, this will mean a short session, and when specifying TTL in the storage, you can specify the value [sessions.cookie_max_age](https://martenframework.com/docs/development/reference/settings#cookie_max_age-1)

For example like this
  https://github.com/martenframework/marten-redis-session/pull/1/commits/8fde4d203a9f2b6a3ef88e124b1b74a89f4a2fdf#diff-6677a4928253a79a8b975ad522b82c587c233b1e0f9d40c9709d9ccd90a44368R68

Thus, this does not imply indefinite storage of the session

Reply:
The case of making the session cookie expire when the browser closes should be implemented with a different method in my opinion. How about adding an `#expires_at_browser_close` method that would set that would result in the `#expires_at` method returning `nil`? This would be more transparent for users (vs letting them specify a 0 expiry).

Reply:
I agree with you.